src/main/java/com/byclosure/webcat/stepdefinitions/BrowsingSteps.java

```diff
@@ -0,0 +1,118 @@
+package com.byclosure.webcat.stepdefinitions;
+
+import com.byclosure.webcat.context.IContext;
+import com.byclosure.webcat.stepdefinitions.helpers.SeleniumInteractions;
+import com.google.inject.Inject;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.apache.commons.validator.routines.UrlValidator;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.io.UnsupportedEncodingException;
+import java.util.List;
+
+public class BrowsingSteps {
+    
+    private final WebDriver driver;
+    private final SeleniumInteractions seleniumInteractions;
+
+    @Inject
+    public BrowsingSteps(WebDriver driver, IContext context){
+        this.driver = driver;
+        this.seleniumInteractions = new SeleniumInteractions(driver, context);
+    }
+    
+    @Given("^(?:|I )am on (.+)$")
+    public void iAmOn(String url) {
+        this.gotToPage(url);
+    }
+    
+    @When("^(?:|I )go to (.+)$")
+    public void iGoTo(String url) {
+        this.gotToPage(url);
+    }
+    
+    private void gotToPage(String url) {
```

Comment:
Couldn't we have 2 annotations (@Given("^(?:|I )am on (.+)$") and @When("^(?:|I )go to (.+)$")) directly into this method? Couldn't we rely in alias?